apps/exchange/lib/exchange/order.ex

```diff
@@ -33,3 +33,9 @@ defmodule Exchange.Order do
           exp_time: integer
         }
 end
+
+defimpl Jason.Encoder, for: Exchange.Order do
+  def encode(value, opts) do
+    Jason.Encode.map(Map.take(value, [:order_id, :trader_id, :side, :price, :size, :initial_size, :type, :exp_time, :acknowledged_at, :modified_at]), opts)
```

Comment:
Vertical lists are usually easier to read. They also produce a better diff when altered and don't require horizontal scrolling / wrapping of text.